database/mysql_db.py

```diff
@@ -186,16 +185,16 @@ def saveAccountInfo(self, username, data):
             data["level"]) + " WHERE user.username = '" + str(username) + "';"
         return querry
 
-    def saveAccountInfoByKey(self, username, key, value):
+    def saveAccountInfoByKey(self, username, key, value) -> Optional[str]:
         self.log_function_name()
         column = self.getColumn(key)
-        if (column is None):
+        if column is None:
```

Comment:
Not a very big fan of this coding style either

Reply:
I need clarification because the reference is ambiguous. I don't know if the coding style you refer to is the one that I replaced or introduced. **Do you prefer using unnecessary parentheses in if statements or prefer to remove them?**

My IDE gives a warning about it so I figured it would just clean things up a bit. Enclosing in parentheses is something we are used to from Java/C but unnecessary in python. I personally prefer to remove them when they don't increase readability.

If you want to maintain the parenthesis I can simply disable those warnings.

Reply:
Lets go with parentheses

Reply:
I'll revert it when I get home